### PR TITLE
Plain `data` field encoding

### LINKED ISSUE #10: Non data base 64 payload submission
## Description

As of now, it's not possible to submit a new print job without sending the data as a base64-encoded string. This can limit some simple scenarios where it is expected to have a simple data string as input.

## Implementation

Make it optional to send either one or the other as job input.

### PR DESCRIPTION
**Pull request by joamag**
Fixes #10

### CODE REVIEW

File: src/colony_print/controllers/node.py [hunk 1]
```diff
@@ -86,14 +87,31 @@ def job_result(self, id, job_id):
...
+        )
+        appier.verify(
+            not (data and data_b64),
+            message="Only one of data or data_b64 fields must be provided",
+            code=400,
+        )
+
         job_id = str(uuid.uuid4())
         name = name or job_id
+        data_b64 = (
```
Comment (hugo-gomes) [inline src/colony_print/controllers/node.py:110]:
```python
if data:
  data_b64 = base64.b64encode(appier.legacy.bytes(data, encoding="utf-8")).decode('utf-8')
```

would argue that this style is more straightforward 😅

Comment (joamag) [reply src/colony_print/controllers/node.py:110]: That is probably right b64encode returns a bytes object and we should offer a str

Comment (joamag) [reply src/colony_print/controllers/node.py:110]: But we can use appier.legacy.str for that i guess

Comment (joamag) [reply src/colony_print/controllers/node.py:110]: Now thinking about it, I believe your option is better :)

Comment (joamag) [reply src/colony_print/controllers/node.py:110]: Done https://github.com/hivesolutions/colony-print/commit/669988376f3f285067fb515e758e5f4ec7393c26

**Pull request merged**